stdlib/@tests/stubtest_allowlists/common.txt

```diff
@@ -12,6 +12,8 @@ _collections_abc.ItemsView.__reversed__
 _collections_abc.KeysView.__reversed__
 _collections_abc.ValuesView.__reversed__
 _ctypes.CFuncPtr  # stubtest erroneously thinks it can't be subclassed
+_?ctypes.Structure.__getattr__  # doesn't exist, but makes things easy if we pretend it does
+_?ctypes.Union.__getattr__  # doesn't exist, but makes things easy if we pretend it does
```

Comment:
Is still correct in the "should be fixed" section?

Reply:
I could go either way on it